test/OlogClientTest.py

```diff
@@ -7,22 +7,29 @@
 @author: shroffk
 '''
 import unittest
+
 from pyOlog import OlogClient
 from pyOlog import Tag, Logbook, Property, LogEntry, Attachment
 from datetime import datetime
 import time
-import os
-from pyOlog.test.OlogDataTypeTest import TestProperty
+from _testConf import _conf
+
 
+def getDefaultTestConfig(arg):
+    if _conf.has_option('DEFAULT', arg):
+        return _conf.get('DEFAULT', arg)
+    else:
+        return None
+        
 class TestCreateClient(unittest.TestCase):
     
     def testCreateClient(self):
         '''
         Simple test to create a ologClient
         '''
-        client = OlogClient(url='http://localhost:8080/Olog')
+        client = OlogClient(url=getDefaultTestConfig('url'))
```

Comment:
I look at this library and compared to the java one it needs significant work, this is just one example of cleanup